### PR TITLE
Issue #3223719: Prepare for wysiwyg_template update

### PR DESCRIPTION
**Pull request by RoSk0**
This is required in order to be able to update to wysiwyg_template:3.0.0-rc2

### CODE REVIEW

**Comment by dieuwedeboer** [comment]
This should just be a one-line change as ckeditor/templates is already included. Just bump the version number up from 4.11.0 to 4.11.1.

I'm also not sure if we need to ship that codesniffer package with the sector-project template either, but happy to consider why it might be desirable to have it present out-of-the-box.

File: composer.json [hunk 1]
```diff
@@ -104,5 +117,8 @@
         "post-update-cmd": [
             "Sector\\composer\\ScriptHandler::createRequiredFiles"
         ]
+    },
+    "require-dev": {
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.7.1"
```
Comment (RoSk0) [inline composer.json]: This is extra , I will remove.

**Comment by RoSk0** [comment]
> This should just be a one-line change as ckeditor/templates is already included. Just bump the version number up from 4.11.0 to 4.11.1.

I don't see `ckeditor/templates` , there is only `ckeditor-plugin/templates`

**Comment by dieuwedeboer** [comment]
> I don't see ckeditor/templates , there is only ckeditor-plugin/templates

Their suggested package name is different to what we use (perhaps following a convention from one of the other plugins), but that doesn't affect anything as long as we are internally consistent.

This one: https://github.com/sparksi/sector-project/blob/9.x/composer.json#L50

Which has a matching block in sector-distribution due to the way composer handles repositories: https://git.drupalcode.org/project/sector/-/blob/9.1.x/composer.json#L130

The actual inclusion happens here: https://git.drupalcode.org/project/sector/-/blob/9.1.x/composer.json#L14

You can see from the download link that they're the same plugin.

**Comment by RoSk0** [comment]
Ok, I see, I was checking only projects names. My dab.

**Pull request closed without merging**